content/en/docs/concepts/services-networking/service.md

```diff
@@ -217,184 +247,51 @@ In the example above, traffic is routed to the single endpoint defined in
 the YAML: `192.0.2.42:9376` (TCP).
 
 {{< note >}}
-The Kubernetes API server does not allow proxying to endpoints that are not mapped to 
-pods. Actions such as `kubectl proxy <service-name>` where the service has no 
-selector will fail due to this constraint. This prevents the Kubernetes API server 
-from being used as a proxy to endpoints the caller may not be authorized to access. 
+The Kubernetes API server does not allow proxying to endpoints that are not mapped to
+pods. Actions such as `kubectl proxy <service-name>` where the service has no
+selector will fail due to this constraint. This prevents the Kubernetes API server
+from being used as a proxy to endpoints the caller may not be authorized to access.
 {{< /note >}}
 
-An ExternalName Service is a special case of Service that does not have
+An `ExternalName` Service is a special case of Service that does not have
 selectors and uses DNS names instead. For more information, see the
 [ExternalName](#externalname) section later in this document.
 
-### Over Capacity Endpoints
+### Over-capacity endpoints
+
 If an Endpoints resource has more than 1000 endpoints then a Kubernetes v1.22 (or later)
 cluster annotates that Endpoints with `endpoints.kubernetes.io/over-capacity: truncated`.
 This annotation indicates that the affected Endpoints object is over capacity and that
 the endpoints controller has truncated the number of endpoints to 1000.
 
+### Endpoints
+
+In the Kubernetes API, an
+[Endpoints](/docs/reference/kubernetes-api/service-resources/endpoints-v1/)
+(the resource kind is plural) defines a list of network endpoints, typically
+referenced by a Service to define which Pods the traffic can be sent to.
+
 ### EndpointSlices
 
 {{< feature-state for_k8s_version="v1.21" state="stable" >}}
 
+(If you're just beginning to learn about Services, it's OK to skip learning
+about the differences between Endpoints and EndpointSlice; the overall effect
+is similar and the control plane synchronizes changes between the two APIs
+where possible).
+
 EndpointSlices are an API resource that can provide a more scalable alternative
-to Endpoints. Although conceptually quite similar to Endpoints, EndpointSlices
-allow for distributing network endpoints across multiple resources. By default,
-an EndpointSlice is considered "full" once it reaches 100 endpoints, at which
+to Endpoints. Although conceptually quite similar to Endpoints, the
+[EndpointSlice API](/docs/reference/kubernetes-api/service-resources/endpoint-slice-v1/)
+allows for distributing the set of network endpoints across multiple resources.
+By default, an EndpointSlice is considered "full" once it reaches 100 endpoints, at which
 point additional EndpointSlices will be created to store any additional
 endpoints.
 
 EndpointSlices provide additional attributes and functionality which is
 described in detail in [EndpointSlices](/docs/concepts/services-networking/endpoint-slices/).
 
-### Application protocol
```

Comment:
@sftim it is hard to me to review since this point, is just a copy and paste to other places or there was some rewording that needs review too?

Reply:
I'd compare the preview to the existing docs, but please also look at commit 2 and onwards as these do have some wording changes.